packages/integrations/mdx/test/mdx-plugins.test.js

```diff
@@ -207,6 +221,23 @@ function rehypeExamplePlugin() {
 	};
 }
 
+function rehypeSvgPlugin() {
+	return (tree) => {
+		tree.children.push({
+			type: 'element',
+			tagName: 'svg',
+			properties: { xmlns:"http://www.w3.org/2000/svg" },
+			children: [
+				{
+					type: 'element',
+					tagName: 'use',
+					properties: { 'xlink:href': '#icon' }
```

Comment:
The correct hast casing is `xLinkHref`

Reply:
Ah right forgot about that
```suggestion
					properties: { 'xLinkHref': '#icon' }
```